.travis.yml

```diff
@@ -28,8 +28,8 @@ before_install:
 addons:
   apt:
     sources:
-      - sourceline: 'deb https://repo.mongodb.org/apt/ubuntu jammy/mongodb-org/7.0 multiverse'
-        key_url: 'https://pgp.mongodb.com/server-7.0.asc'
+      - sourceline: 'deb https://repo.mongodb.org/apt/ubuntu jammy/mongodb-org/6.0 multiverse'
```

Comment:
I assume this is for consistency with other services?

Reply:
Yes.

My understanding is, that even though we were using the 7.0 repo previously, we were using 6.x releases of MongoDB, which matches current prod. At some point the 6.x releases of MongoDB stopped being packaged in the 7.0 repo (I don't know why they ever were), and so we've had to make this adjustment. This isn't the first package to see this change, we're just updating them as we find them.